dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -490,6 +492,7 @@ export const renderElement = ({
 					editionId={editionId}
 					RenderArticleElement={RenderArticleElement}
 					isLastElement={index === totalElements - 1}
+					sectioned={!!isSectionedMiniProfilesArticle}
```

Comment:
Does `isSectionedMiniProfilesArticle` need to be coerced to a boolean if it's already a boolean?